frontend/benefit/applicant/src/components/applications/applicationList/listItem/ListItem.tsx

```diff
@@ -1,3 +1,5 @@
+import 'react-loading-skeleton/dist/skeleton.css';
```

Comment:
Placing this import here is a bit curious. My first instinct would be to put it where all other library CSS imports are, to `_app.tsx`; is there a reason you'd put it specifically here instead, other than that the skeleton is relevant to this component?

Reply:
Yes, maybe change that there. I was thinking NextJS is smart enough to do some sort of splitting and lazy load of assets but then again 99% of cases would download that CSS anyway 🤷